### PR TITLE
zcash_client_sqlite: Return a balance for every account from `get_wallet_summary`

### LINKED ISSUE #948: zcash_client_sqlite: Ensure that `WalletSummary` has a balance for every account in the wallet.

### PR DESCRIPTION
**Pull request by nuttycom**
Fixes #948

### CODE REVIEW

File: zcash_client_sqlite/src/wallet.rs [hunk 1]
```diff
@@ -689,27 +697,17 @@ pub(crate) fn get_wallet_summary(
...
-                    + spendable_value)
-                    .expect("Spendable value cannot overflow");
-                bal.sapling_balance.change_pending_confirmation =
-                    (bal.sapling_balance.change_pending_confirmation + change_pending_confirmation)
-                        .expect("Pending change value cannot overflow");
-                bal.sapling_balance.value_pending_spendability =
-                    (bal.sapling_balance.value_pending_spendability + value_pending_spendability)
-                        .expect("Value pending spendability cannot overflow");
-            })
-            .or_insert(AccountBalance {
```
Comment (str4d) [inline zcash_client_sqlite/src/wallet.rs:705]: Non-blocking: we could probably also remove the `or_insert` from the transparent part.

**Review by str4d — APPROVED** [review]
utACK 0771cb26d818263165e614415a24ddd9b441bfd4

File: zcash_client_sqlite/src/wallet.rs (hunk 1 above)
Comment (nuttycom) [reply zcash_client_sqlite/src/wallet.rs:705]: Fixed.

**Review by str4d — APPROVED** [review]
utACK 2131cd547e56b9de836d431c72890d41e7851837

**Pull request merged**